pkg/cmd/kind/root.go

```diff
@@ -56,7 +56,8 @@ func NewCommand(logger log.Logger, streams cmd.IOStreams) *cobra.Command {
 		SilenceErrors: true,
 		Version:       version.Version(),
 	}
-	cmd.SetOutput(streams.ErrOut)
+	cmd.SetOut(streams.Out)
+	cmd.SetErr(streams.ErrOut)
```

Comment:
Is this the only breaking cobra change?

Reply:
yes, internally fish completion will run `kind __complete <args>` to capture the output used for completion. Hence, the current `cmd.SetOutput(streams.ErrOut)` will make stdout empty.

Reply:
er sorry, to be clear: I meant with the large cobra version change specifically

Reply:
I didn't realize we were so far behind (!)

Reply:
Oh it's not much changes from 0.0.5, only bug fixes and new feature for go custom completion. Actually, 0.0.5 is not so far befind, the maintainer of cobra feels it's good time to make it 1.0.0 version, the version before is 0.0.7.

FYI. The above  `SetOuput` is still available, it's just deprecated. As I need to change output stream to std.Out for fish completion to work, I take this chance to use `SetOut` and `SetErr` methods as well.

Reply:
thanks!